lib/rust/ensogl/core/src/display/object.rs

```diff
@@ -23,3 +28,138 @@ pub mod traits {
     pub use super::Object as TRAIT_Object;
     pub use super::ObjectOps as TRAIT_ObjectOps;
 }
+
+
+
+// =============================
+// === InstanceWithSublayers ===
+// =============================
+
+/// A wrapper around [`Instance`] with attached sublayers. It is used for
+/// dynamic creation and managing of additional on-demand sublayers needed for various UI
+/// components.
```

Comment:
No one reading this documentation would understand what a "sublayer" is. Please link it to the correct place in other modules in the docs.